config/environments/production.rb

```diff
@@ -81,12 +81,14 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 
-  # config.action_mailer.delivery_method = :postmark
+  config.action_mailer.delivery_method = :smtp
```

Comment:
I wouldn't say this was the default, since it was commented out. My thought is it's possible that it won't affect anything because it was absent before. But might need to make it configurable based on an env variable 🤔